stock_inventory_revaluation/models/stock_inventory_revaluation.py

```diff
@@ -0,0 +1,407 @@
+                            help='Displays the previous cost of the '
+                                 'product.',
+                            digits=dp.get_precision('Product Price'),
+                            readonly=True)
+
+    current_cost = fields.Float('Current cost',
+                                help='Displays the current cost of the '
+                                     'product.',
+                                digits=dp.get_precision('Product Price'),
+                                compute="_calc_current_cost",
+                                readonly=True)
+
+    new_cost = fields.Float('New cost',
+                            help="Enter the new cost you wish to assign to "
+                                 "the product. Relevant only when the "
+                                 "selected revaluation type is Price Change.",
+                            digits=dp.get_precision('Product Price'))
+
+    current_value = fields.Float('Current value',
+                                 help='Displays the current value of the '
+                                      'product.',
+                                 digits=dp.get_precision('Account'),
+                                 compute="_calc_product_template_value",
+                                 readonly=True)
+
+    old_value = fields.Float('Old value',
+                             help='Displays the current value of the product.',
+                             digits=dp.get_precision('Account'),
+                             readonly=True)
+
+    new_value = fields.Float('Credit/Debit amount',
+                             help="Enter the amount you wish to credit or "
+                                  "debit from the current inventory value of "
+                                  "the item. Enter credit as a negative value."
+                                  "Relevant only if the selected revaluation "
+                                  "type is Inventory Credit/Debit.",
+                             digits=dp.get_precision('Account'))
+
+    qty_available = fields.Float(
+        'Quantity On Hand', compute='_get_product_template_qty',
+        digits_compute=dp.get_precision('Product Unit of Measure'))
+
+    increase_account_id = fields.Many2one(
+        'account.account', 'Increase Account',
+        help="Define the G/L accounts to be used as the balancing account in "
+             "the transaction created by the revaluation. The Increase "
+             "Account is used when the inventory value is increased due to "
+             "the revaluation.")
+
+    decrease_account_id = fields.Many2one(
+        'account.account', 'Decrease Account',
+        help="Define the G/L accounts to be used as the balancing account in "
+             "the transaction created by the revaluation. The Decrease "
+             "Account is used when the inventory value is decreased.")
+
+    move_id = fields.Many2one('account.move', 'Account move', readonly=True,
```

Comment:
It would be nice to name the field `account_move_id` (we're dealing with the inventory so some developers could misunderstand "move" as "stock move")